integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
```

Comment:
Should these values be defined here, or in a subclass specific to SAP SuccessFactors? If we integrated another channel (please, not for a while), then `GRADE_PASSING` might be `127`, `GRADE_FAILING` might be `0`, and `GRADE_INCOMPLETE` might be `-1`. It's only in the case of SuccessFactors that we actually know that we can send off a free-form string, right?